packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -16877,6 +16877,80 @@ void main() {
     variant: const TargetPlatformVariant(<TargetPlatform>{TargetPlatform.iOS}),
     skip: kIsWeb, // [intended]
   );
+
+  testWidgets('onTapUpOutside is called upon tap up outside', (WidgetTester tester) async {
+    int tapOutsideCount = 0;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Center(
+            child: Column(
+              children: <Widget>[
+                const Text('Outside'),
+                EditableText(
+                  autofocus: true,
+                  controller: controller,
+                  focusNode: focusNode,
+                  style: textStyle,
+                  cursorColor: Colors.blue,
+                  backgroundCursorColor: Colors.grey,
+                  onTapUpOutside: (PointerEvent event) {
+                    tapOutsideCount += 1;
+                  },
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+    await tester.pump(); // Wait for autofocus to take effect.
+
+    expect(tapOutsideCount, 0);
+    await tester.tap(find.byType(EditableText));
+    await tester.tap(find.text('Outside'));
+    await tester.tap(find.text('Outside'));
+    await tester.tap(find.text('Outside'));
+    expect(tapOutsideCount, 3);
+  });
+
+  // Regression test for https://github.com/flutter/flutter/issues/162573
+  testWidgets('onTapUpOutside is not called upon tap up outside when field is not focused', (
```

Comment:
The equivalent test for `onTapOutside` is located in [text_form_field_test.dart#L741](https://github.com/Mairramer/flutter/blob/b4a81cb6da84b7658b984922fb7056920058a14e/packages/flutter/test/material/text_form_field_test.dart#L741), but in my opinion it should have been located in this file instead. That's why I added this test here. Let me know if I should move it to `text_form_field_test.dart` instead.

Reply:
I think this test is good in `editable_text_test.dart`. If you'd like you can also transfer the equivalent `onTapOutside` test from `text_form_field_test.dart` to `editable_text_test.dart`, I think it fits better in here.